authors/apps/profiles/urls.py

```diff
@@ -1,11 +1,24 @@
 from django.urls import path
 
 from authors.apps.profiles.views import ProfileRetrieveAPIView
+from authors.apps.profiles.views import FollowUserRetrieveAPIView
```

Comment:
PEP8: I think it would have been better to use one import statement to import all four classes

Reply:
Thanks Lenny, according to [PEP8](https://www.python.org/dev/peps/pep-0008/) the imports should be separated without a comma.

```
Imports should usually be on separate lines:

Yes: import os
     import sys

No:  import sys, os
```